From 8acf1ebc81b61fc7e3e1e02ba68100094577c19c Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Fri, 19 May 2023 17:19:03 +0800
Subject: [PATCH 1/1] fansensor: add compatible string for ast2600 tach

Add `aspeed,ast2600-pwm-tach` as compatible string for ast2600 pwm tach driver.

This commit relies on the kernel patch below.
link: https://lore.kernel.org/all/20230918064111.2221594-1-billy_tsai@aspeedtech.com/

Tested on Bletchley.

Change-Id: Ia2fcf6cd8856edfb15871a3b68a8ffbc13b8025d
Signed-off-by: Potin Lai <potin.lai@quantatw.com>
---
 src/FanMain.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/FanMain.cpp b/src/FanMain.cpp
index 7cfb533..7d0ac38 100644
--- a/src/FanMain.cpp
+++ b/src/FanMain.cpp
@@ -66,6 +66,7 @@ std::optional<RedundancySensor> systemRedundancy;
 static const std::map<std::string, FanTypes> compatibleFanTypes = {
     {"aspeed,ast2400-pwm-tacho", FanTypes::aspeed},
     {"aspeed,ast2500-pwm-tacho", FanTypes::aspeed},
+    {"aspeed,ast2600-pwm-tach", FanTypes::aspeed},
     {"nuvoton,npcm750-pwm-fan", FanTypes::nuvoton},
     {"hpe,gxp-fan-ctrl", FanTypes::hpe}
     // add compatible string here for new fan type
-- 
2.31.1

